browserstack.yml

```diff
@@ -6,6 +6,8 @@
 userName: BROWSERSTACK_USERNAME
 accessKey: BROWSERSTACK_ACCESS_KEY
 
+framework: selenide-testng
```

Comment:
let's keep testng, selenide-testng will break / sdk changes will be required. Please confirm